hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStream.java

```diff
@@ -46,6 +50,7 @@
 public class AbfsInputStream extends FSInputStream implements CanUnbuffer,
         StreamCapabilities {
   private static final Logger LOG = LoggerFactory.getLogger(AbfsInputStream.class);
+  public static final int FOOTER_SIZE = 16 * ONE_KB;
```

Comment:
Add a comment that footer size is set to qualify for both ORC and parquet files.

Reply:
Done